analysers/analyser_merge_milestone_FR_metropole.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, config, logger = None):
             "Bornage du réseau routier national",
             CSV(Source(attribution = "data.gouv.fr:Ministère de la Transition écologique et solidaire", millesime = "01/2020",
                     fileUrl = "https://www.data.gouv.fr/fr/datasets/r/7de08adc-74ae-4e62-8967-6f559ff6cbed"),
-                    separator = "\t"),
+                    separator = ","),
```

Comment:
Comma is the default, you can just remove the separator param.

Reply:
thanks